src/surface.rs

```diff
@@ -0,0 +1,53 @@
+//! Utility functions for creating dpi aware wayland surfaces.
+
+use std::sync::{Arc, Mutex};
+use wayland_client::Proxy;
+use wayland_client::protocol::wl_surface;
+use wayland_client::protocol::wl_compositor::{RequestsTrait as CompositorRequest};
+use crate::env::Environment;
```

Comment:
SCTK tries to remain compatible with older versions of rust, as such we can't use features like `crate::`